File: C/1046-tempo-de-jogo.c
#include <stdio.h>
 
int main() {
 
    int inicio, final, result;
    
    scanf("%d %d",&inicio,&final);
    
    if (inicio>=final)
    {
       result = (24-inicio)+final;
       printf("O JOGO DUROU %d HORA(S)\n",result);
    }
    else
    {
       result = final-inicio;
       printf("O JOGO DUROU %d HORA(S)\n",result);
    }
 
    return 0;
}

File: PYTHON/Iniciante/1046-tempo-de-jogo.py
h_inicial, h_final = map(int, input().split(" "))

if h_inicial >= h_final:
   duracao = (24-h_inicial) + h_final
else:
   duracao = h_final - h_inicial

print("O JOGO DUROU {} HORA(S)".format(duracao))